modules/runner/src/integrationTest/java/org/apache/ignite/internal/calcite/ItDataTypesTest.java

```diff
@@ -22,20 +22,18 @@
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Collectors;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 /**
  * Test SQL data types.
  */
-@Disabled("https://issues.apache.org/jira/browse/IGNITE-15107")
```

Comment:
This test should be disabled with https://issues.apache.org/jira/browse/IGNITE-15655

Reply:
check comment upper, seems all ok here.